.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+# Pull Request Template
+
+## Description
+
+Please include a summary of the change and which issue is fixed.
+
+Fixes # (youtrack issue)
```

Comment:
Does this make sense in the public library? Our youtrack is private, so maybe it should rather reference a github issue?